public/app/plugins/datasource/grafana-azure-monitor-datasource/components/LogsQueryEditor/QueryField.tsx

```diff
@@ -0,0 +1,32 @@
+import { CodeEditor } from '@grafana/ui';
+import React, { useCallback } from 'react';
+import { AzureQueryEditorFieldProps } from '../../types';
+
+const QueryField: React.FC<AzureQueryEditorFieldProps> = ({ query, onQueryChange }) => {
+  const onChange = useCallback(
+    (newQuery: string) => {
+      onQueryChange({
+        ...query,
+        azureLogAnalytics: {
+          ...query.azureLogAnalytics,
+          query: newQuery,
+        },
+      });
+    },
+    [onQueryChange, query]
+  );
+
+  return (
+    <CodeEditor
+      value={query.azureLogAnalytics.query}
+      language="kql"
+      height={200}
+      width="100%"
+      showMiniMap={false}
```

Comment:
Do you need to set this prop?

Reply:
Need? No. Want? yes!

The minimap seems distracting and offers no benefit for a small code editor like this. IMHO it should default to off for all `<CodeEditor/>`s, but that's not for here :)